example/example_resourcemanager_test.go

```diff
@@ -0,0 +1,114 @@
+package example
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/oracle/oci-go-sdk/common"
+	"github.com/oracle/oci-go-sdk/example/helpers"
+	"github.com/oracle/oci-go-sdk/resourcemanager"
+)
+
+func ExampleResourceManager() {
+	provider := common.DefaultConfigProvider()
```

Comment:
in general, when add samples, we would recommend to add comments as document for people to have a general idea about the scope of this sample before read into the detail code.